docs/guides/mysql/schema-manager/deploy-mysqldatabase/index.md

```diff
@@ -0,0 +1,428 @@
+---
+title: Deploy MySQLDatabase
+menu:
+  docs_{{ .version }}:
+    identifier: deploy-mysqldatabase
+    name: Deploy MySQLDatabase
+    parent: guides-mysql-schema-manager
+    weight: 15
+menu_name: docs_{{ .version }}
+section_menu_id: guides
+---
+
+> New to KubeDB? Please start [here](/docs/README.md).
+
+{{< notice type="warning" message="This is an Enterprise-only feature. Please install [KubeDB Enterprise Edition](/docs/setup/install/enterprise.md) to try this feature." >}}
+
+# Create Database with MySQL Schema Manager
+
+This guide will show you how to create database with MySQL Schema Manager using `KubeDB Enterprise Operator`.
+
+## Before You Begin
+
+- At first, you need to have a Kubernetes cluster, and the `kubectl` command-line tool must be configured to communicate with your cluster. If you do not already have a cluster, you can create one by using [kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+
+- Install `KubeDB Enterprise Operator` in your cluster following the steps [here](https://kubedb.com/docs/latest/setup/install/enterprise/).
+- Install `KubeVault Enterprise Operator` in your cluster following the steps [here](https://kubevault.com/docs/latest/setup/install/enterprise/).
+
+- You should be familiar with the following `KubeDB` and `KubeVault` concepts:
+  - [MySQL](/docs/guides/mysql/concepts/database/index.md)
+  - [MySQLDatabase](/docs/guides/mysql/concepts/mysqldatabase/index.md)
+  - [Schema Manager Overview](/docs/guides/mysql/schema-manager/overview/index.md)
+  - [KubeVault Overview](https://kubevault.com/docs/latest/concepts/overview/)
+
+To keep everything isolated, we are going to use a separate namespace called `demo` throughout this tutorial.
+
+```bash
+$ kubectl create ns demo
+namespace/demo created
+```
+
+> **Note:** YAML files used in this tutorial are stored in [docs/guides/mysql/schema-manager/deploy-mysqldatabase/yamls](https://github.com/kubedb/docs/tree/{{< param "info.version" >}}/docs/guides/mysql/schema-manager/deploy-mysqldatabase/yamls) directory of [kubedb/doc](https://github.com/kubedb/docs) repository.
+
+## Deploy MySQL Server and Vault Server 
+
+Firstly, we are going to deploy a `MySQL Server` by using `KubeDB` operator. Also, we are deploying a `Vault Server` using `KubeVault` Operator.
+
+### Deploy MySQL Server
+
+In this section, we are going to deploy a MySQL Server. Below is the YAML of the `MySQL` CR that we are going to create,
```

Comment:
`let's deploy a MySQL server using the following yaml`

Reply:
or 
`let's deploy a MySQL server using the following yaml. which will create a MySQL standalone instance of MySQL version 8.0.29`